src/async_fuse/memfs/cache/storage_manager.rs

```diff
@@ -928,11 +928,10 @@ mod tests {
                 mtime = storage.store(0, offset, BLOCK_CONTENT, mtime).await;
             }
 
-            // Wait for all blocks being flushed
-            tokio::time::sleep(Duration::from_millis(900)).await;
-
             let (latency, ()) = elapsed!(storage.flush(0).await);
-            assert!(latency.as_millis() < 10, "latency = {latency:?}");
+            // All blocks are flushed concurrently, therefore the latency of flushing
+            // should be a single operation latency (100ms) + some extra costs (< 10ms)
+            assert!(latency.as_millis() < 110, "latency = {latency:?}");
```

Comment:
hard code the latency is not good. Is there any constant definition?

Reply:
Yes, it's not good. I will fix it later.